includes/core.php

```diff
@@ -286,18 +279,32 @@ public function get_challengeurl()
 
   public function get_translations($language = null)
   {
-    if ($language === null) {
-      $language = $this->get_language();
+    $originalLanguage = null;
+
+    if ($language !== null) {
+      $originalLanguage = get_locale();
+      switch_to_locale($language);
     }
-    if ($language === "auto") {
-      // Get the current locale
-      $language = get_locale();
-      if (!isset(ALTCHA_TRANSLATIONS[$language])) {
-        // Wordpress uses full codes such as `fr_FR`; if no translation is found, try short version `fr`
-        $language = substr($language, 0, 2);
-      }
+
+    $ALTCHA_WEBSITE = constant('ALTCHA_WEBSITE');
+    $translations = array(
+      "error" => __("Verification failed. Try again later.", "altcha-spam-protection"),
+      "footer" => sprintf(
+        __("Protected by %sALTCHA%s", "altcha-spam-protection"),
```

Comment:
It just occurred to me that since ALTCHA is sort of a brand name, which shouldn't be changed, this string could be simplified to `"Protected by %s"`, placing the name inside the placeable as well.